packages/cpt-ui/components/EpsHeader.tsx

```diff
@@ -10,20 +10,33 @@ import {
   HEADER_CONFIRM_ROLE_TARGET,
   HEADER_CHANGE_ROLE_BUTTON,
   HEADER_CHANGE_ROLE_TARGET,
-  HEADER_SELECT_YOUR_ROLE_BUTTON,
-  HEADER_SELECT_YOUR_ROLE_TARGET
+  HEADER_SELECT_YOUR_ROLE_TARGET,
+  HEADER_SELECT_YOUR_ROLE_BUTTON
 } from "@/constants/ui-strings/HeaderStrings";
 
 import { AuthContext } from "@/context/AuthProvider";
+import { useAccess } from '@/context/AccessProvider';
+
 import { EpsLogoutModal } from "@/components/EpsLogoutModal";
 
 export default function EpsHeader() {
   const router = useRouter();
   const pathname = usePathname();
   const auth = useContext(AuthContext);
-
+  const accessContext = useAccess();
```

Comment:
For some reason, the way this context is being loaded causes it to re-initialise whenever a component spins up. I've checked and this happens wherever I put the context - why!?!?